apps/discovery_api/test/unit/test_helper.exs

```diff
@@ -1,2 +1,2 @@
-ExUnit.start(exclude: [:skip])
+ExUnit.start(exclude: [:skip], capture_log: true)
```

Comment:
Should we use this on a test by test basis instead of the entire unit suite?

https://elixirschool.com/blog/til-capture-log-in-exunit-tests